kolibri/core/assets/src/composables/useNow.js

```diff
@@ -0,0 +1,21 @@
+import { now as getNow } from 'kolibri.utils.serverClock';
+
+import { ref, onMounted, onUnmounted } from 'kolibri.lib.vueCompositionApi';
+
+export default function useNow(interval = 10000) {
```

Comment:
To justify the composable, we could use this in the ElapsedTime component as well?

Reply:
I created a follow-up issue for this in #12114